iam-proxy-italia-project/backends/ciesaml2.py

```diff
@@ -145,6 +155,8 @@ def _metadata_contact_person(self, metadata, conf):
         ###################
 
     def _metadata_endpoint(self, context):
+        logger.debug(f"Entering method: {inspect.getframeinfo(
+            inspect.currentframe()).function}. Params [context: {context}]")
```

Comment:
please in general do indentations like the following suggested one

```suggestion
        logger.debug(
            f"Entering method: {inspect.getframeinfo(inspect.currentframe()).function}. Params [context: {context}]"
        )
```